tests/com.redhat.fabric8analytics.lsp.eclipse.ui.itests/src/com/redhat/fabric8analytics/lsp/eclipse/ui/itests/dialogs/OpenshiftServicesPreferenceDialog.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2007-2017 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v 1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.fabric8analytics.lsp.eclipse.ui.itests.dialogs;
+
+import static org.junit.Assert.assertTrue;
+
+import org.eclipse.reddeer.common.matcher.RegexMatcher;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.core.matcher.WithTextMatcher;
+import org.eclipse.reddeer.swt.impl.button.CheckBox;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.eclipse.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
+import org.eclipse.reddeer.workbench.workbenchmenu.WorkbenchMenuPreferencesDialog;
+import org.jboss.tools.openshift.reddeer.preference.page.OpenShifIOPreferencePage;
+
+import com.redhat.fabric8analytics.lsp.eclipse.ui.itests.pages.OpenshiftServicesPreferencePage;
+
+public class OpenshiftServicesPreferenceDialog extends WorkbenchMenuPreferencesDialog {
```

Comment:
this class is not needed at all. You can just use PrefernecesDialog and all the methods (enable/disable server) should be moved to OpenshiftServicesPreferencePage